infrastructure/resource_group_level/modules/vnet.bicep

```diff
@@ -7,10 +7,17 @@ param applicationGatewayCidr string
 
 param appServicePlanName string
 
+// TODO:DTFS-6422 check route table config isn't needed and happens automatically from the route table setup.
 param routeTableId string
 param networkSecurityGroupId string
-// TODO:DTFS-6422 Note that peeringAddressSpace doesn't seem to appear in the old IaC scripts
+
+// TODO:DTFS-6422 Note that the staging name is: "tfs-test-vnet_vnet-ukef-uks". We should ensure we start with that name as an override.
+param peeringVnetName string = 'tfs-${environment}-vnet_vnet-ukef-uks'
```

Comment:
I think it is best to start with the misnaming on staging, and then change it later.